modules/persistent-storage-csi-group-snapshot-create-admin.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-group-snapshots-create-admin_{context}"]
+= Creating a volume group snapshot class
+
+Before you can create volume group snapshots, the cluster administrator needs to create a `VolumeGroupSnapshotClass`. 
+
+This object describes how volume group snapshots should be created, including the driver information, the deletion policy, etc.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster with administrator privileges.
+
+.Procedure
+
+To create a `VolumeGroupSnapshotClass`:
+
+. Create a 'VolumeGroupSnapshotClass' by running the following command and using the example YAML file:
++
+[source,terminal]
+----
+$ oc create -f <volume-group-snapshot-class-filename>.yaml
+----
++
+. Example volume group snapshot class YAML file
+[source, yaml]
+----
+apiVersion: volumegroup.storage.k8s.io/v1alpha1
+kind: VolumeGroupSnapshotClass <1>
+metadata:
+  name: <volume-group-snapshot-class-name> <2>
+spec:
+  parameters:
```

Comment:
Yes. I know I need a better example file to show parameters. Let me know if you have one!

Reply:
Thanks, I've used your file.